src/lib/yaraparse.py

```diff
@@ -148,11 +154,30 @@ def ursify_plain_string(string: PlainString) -> UrsaExpression:
         return ursa_ascii
 
 
+def ursify_xor_string(string: PlainString) -> UrsaExpression:
+    text_ascii = string.pure_text
+    xored_strings: List[UrsaExpression] = []
+
+    for xor_key in range(256):
+        xored_ascii = xor(text_ascii, bytes([xor_key]))
+        xored_wide = bytes(x ^ xor_key for y in text_ascii for x in [y, 0])
+
+        if string.is_ascii:
+            xored_strings.append(UrsaExpression(f"{{{xored_ascii.hex()}}}"))
```

Comment:
created an issue to refactor it later #99 (also it's a good candidate for a first issue for someone)